doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1132 @@
+  * Updated minimal libc to print stderr just like stdout.
+  * Added an ``abort()`` function to minimal libc.
+  * Updated the ring buffer to allow using the full buffer capacity instead of forcing an empty slot.
+  * Added a :c:macro:`CLAMP` macro.
+  * Added a feature for post-mortem analysis to the tracing library.
+
+* Samples:
+
+  * Added :ref:`zephyr:nrf-ieee802154-rpmsg-sample`.
+  * Added :ref:`zephyr:cloud-tagoio-http-post-sample`.
+  * Added :ref:`zephyr:civetweb-websocket-server-sample`.
+  * :ref:`zephyr:led_ws2812_sample`: Updated to force SPIM on nRF52 DK.
+  * :ref:`zephyr:cfb_custom_fonts`: Added support for ssd1306fb.
+  * :ref:`zephyr:gsm-modem-sample`: Added suspend/resume shell commands.
+  * :ref:`zephyr:updatehub_fota_sample`: Added support for |BLE| IPSP, 802.15.4, modem, and Wi-Fi.
+
+* Logging:
+
+  * Added STP transport and raw data output support for systrace.
+
+* Modules:
+
+  * Introduced a :option:`CONFIG_MBEDTLS_MEMORY_DEBUG` option for mbedtls.
+  * Updated LVGL to v7.6.1.
+  * Updated libmetal and openamp to v2020.10.
+  * Updated nrfx in hal-nordic to version 2.4.0.
+  * Updated the Trusted Firmware-M (TF-M) module to v1.2.0.
+  * Moved the nrfx glue code from the hal_nordic module repository to the main Zephyr repository.
+  * Updated the Trusted Firmware-M (TF-M) module to include support for the nRF5340 and nRF9160 platforms.
+
+* Other:
+
+  * Renamed the ``sanitycheck`` script to ``twister``.
+  * Added initial LoRaWAN support.
+  * Updated ``west flash`` support for ``nrfjprog`` to fail if a HEX file has UICR data and ``--erase`` was not specified.
+  * Added an API to correlate system time with external time sources (see :ref:`zephyr:timeutil_api`).
+
+* Power management:
+
+  * Overhauled the naming and did some general cleanup.
+  * Added a notifier API to register an object to receive notifications when the system changes power state.
+
+* Shell:
+
+  * Updated documentation.
+  * Optimized the tab feature and the select command.
+  * Enhanced and improved the help command.
+
+* Toolchain:
+
+  * Added initial support for LLVM/Clang (version 10, on the x86 architecture).
+  * Added the environment variable ``LLVM_TOOLCHAIN_PATH`` for locating the LLVM toolchain.
+
+* USB:
+
+  * Fixed the handling of zero-length packet (ZLP) in the nRF USB Device Controller Driver.
+  * Changed the USB DFU wait delay to be configurable with Kconfig (:option:`CONFIG_USB_DFU_WAIT_DELAY_MS`).
+
+Additions specific to |NCS|
++++++++++++++++++++++++++++
+
+The following list contains |NCS| specific additions:
+
+* Added support for the |NCS|'s :ref:`partition_manager`, which can be used for flash partitioning.
+* Added the following network socket and address extensions to the :ref:`zephyr:bsd_sockets_interface` interface to support the functionality provided by the BSD library:
+
+  * AF_LTE
```

Comment:
This section seems to be a bit rusty, some things are missing, I think we could simply use the summary from the commit message which is up to date (with some minor tweaks):
```
  * AF_LTE family
  * NPROTO_AT protocol
  * NPROTO_PDN protocol to be used in conjuction with AF_LTE
  * NPROTO_DFU protocol to be used in conjuction with AF_LOCAL
  * SOCK_MGMT socket type, used in conjunction with AF_LTE
  * SOL_PDN protocol level and associated socket option values: SO_PDN_CONTEXT_ID option for PDN sockets, SO_PDN_STATE option for PDN sockets to retrieve the state of the PDN connection
  * SOL_DFU protocol level and associated socket options. This includes a SO_DFU_ERROR socket option for DFU socket can be used when an operation on a DFU socket returns -ENOEXEC, indicating that the modem has rejected the operation, to retrieve the reason for the error
  * TLS_SESSION_CACHE socket option for TLS session caching
  * SO_BINDTODEVICE socket option
  * SO_SNDTIMEO socket option
  * SO_SILENCE_ALL to disable/enable all replies to unexpected traffics
  * SO_IP_ECHO_REPLY to disable/enable replies to IPv4 ICMPs
  * SO_IPV6_ECHO_REPLY to disable/enable replies to IPv6 ICMPs
  * MSG_TRUNC socket flag
  * MSG_WAITALL socket flag required to support the corresponding NRF counterpart flag, for translation within the offloading interface
```

Reply:
Updated.